### DISCUSSION TITLE
Is there any way to customize fonts, etc per notification category?

### DISCUSSION THREAD

**Discussion by rbutoi** [ErikReider/SwayNotificationCenter#197] [Q&A] [answered]
As in `mako`'s [CRITERIA](https://man.archlinux.org/man/mako.5.en#CRITERIA). I'd like to make certain notifications use monospaced font, is this possible?

**Comment by ErikReider** [comment, marked as answer]
Sadly CSS is kind of limited in that regard... I could be wrong though
